synapse/storage/_base.py

```diff
@@ -1577,39 +1582,51 @@ def _simple_select_list_paginate_txn(
         cls,
         txn,
         table,
-        keyvalues,
         orderby,
         start,
         limit,
         retcols,
+        filters=None,
+        keyvalues=None,
         order_direction="ASC",
     ):
         """
         Executes a SELECT query on the named table with start and limit,
         of row numbers, which may return zero or number of rows from start to limit,
         returning the result as a list of dicts.
+        Use filters to search attributes using SQL regular expressions and/or keyvalues
```

Comment:
backticks help show that we are referring to a specific parameter, rather than "filters" as a general concept

```suggestion

        Use `filters` to search attributes using SQL wildcards and/or `keyvalues`
```